django/db/models/enums.py

```diff
@@ -49,7 +49,7 @@ def validate(cls, value):
         try:
             cls(value)
         except ValueError:
-            raise exceptions.ValidationError(_(f'Invalid choice: {cls.__name__}({value!r})'))
+            raise exceptions.ValidationError(_('Invalid choice: %s(%r)' % (cls.__name__, value)))
```

Comment:
`%s(%r)` -> `%s (%r)`

Reply:
Hmm, could do. I think that it was `%s(%r)` because `Fruit('APPLE')` is equivalent to `Fruit.APPLE`.

Otherwise it could have been rewritten as `'Invalid choice for %s: %r.'`

Reply:
Ahh OK, so maybe `%s.%s`.

Reply:
Sorry - I confused here. It is trying to show that passing an invalid member name to the enum is what has caused the validation error. But I guess we are trying to hide that as an implementation detail, so we could go for my alternative.

I'm going to investigate using ``__contains__``, however, as this was an attempt to provide an alternate way to work around ``'APPLE' in Fruit``.